requirements/pip.txt

```diff
@@ -18,25 +18,25 @@ future==0.16.0
 django-tastypie==0.13.0
 
 django-guardian==1.4.9
-django-extensions==2.0.7
+django-extensions==2.1.0
 
 # djangorestframework 3.7.x drops support for django 1.9.x
 djangorestframework==3.6.4
 
 # Filtering for the REST API
-django-filter==1.1.0
+django-filter==2.0.0
```

Comment:
We may need to check the migration guide to 2.0

https://django-filter.readthedocs.io/en/master/guide/migration.html#migrating-to-2-0

Reply:
We don't use this for much so it should be fairly easy to test whether this works.

Reply:
I don't believe we can upgrade this one. Firstly, `django-filter==2.0.0` [requires Django 1.11](https://github.com/carltongibson/django-filter/blob/2.0.0/setup.py#L58) which we haven't upgraded to yet. Once we do that upgrade though, this migration should be fairly straight-forward. It looks like they changed the name of `filter_fields` => `filterset_fields`. We use that [in the API](https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/restapi/views/model_views.py)

Reply:
I made a mistake when pushing my changes and I lost part of my work. There should be a note here saying what you just mentioned.

I'm sorry about that. I will re-add it.